src/modules/CreateDrops/TicketDrop/CreateTicketDropSummary.tsx

```diff
@@ -0,0 +1,20 @@
+import { DropSummary } from '../DropSummary';
+
+import { useCreateTicketDropContext } from './CreateTicketDropContext/CreateTicketDropContext';
+
+export const CreateTicketDropSummary = () => {
+  const contextData = useCreateTicketDropContext();
```

Comment:
can we destructure this? so it becomes 
```
const {getSummaryData, getPaymentData, createLinksSWR} = ...
```

Reply:
Destructured the code here [696872e](https://github.com/keypom/keypom-airfoil/pull/68/commits/696872e4f66e03e8ebb30490cc19138eb67947c1)